aw_watcher_afk/config.py

```diff
@@ -3,13 +3,13 @@
 
 default_settings = {
     "timeout": "180",
-    "update_interval": "30",
```

Comment:
Just enforcing consistency with the aw-watcher-window config. If we want interval we should change it in both places.